### DISCUSSION TITLE
Can you share the source package file of rpm? What about hosting a copr for Fedora and a PPA for Debian?

### DISCUSSION THREAD

**Discussion by rashidi77** [OpenBangla/OpenBangla-Keyboard#285] [Ideas]
1. I want to see the rpm-source files of the last stable binary release. Is it possible to share the source package files for Fedora RPMs?
2. Please consider hosting a PPA and a Copr for Debian (based distros) and Fedora. It'll be helpful to easily install and maintain updates for end-users. You can even push bug-fixes and security updates using those repos.

**Comment by mominul** [comment]
Actually, they are generated on the fly by cpack (an accompanying tool comes with cmake). You can look at the configurations [here](https://github.com/OpenBangla/OpenBangla-Keyboard/blob/b70ce3237afc9e7edc5251c41fc2b38d021a5ea0/CMakeLists.txt#L80).

Inclusion of OBK directly into Ubuntu repository [was in progress](https://github.com/OpenBangla/OpenBangla-Keyboard/discussions/207) but the new release kept being delayed so I could not contact him again.

I am really eager about having a copr repo for fedora based system, but I am not familiar with manually creating rpm. So it's always getting delayed. Are you interested about handling RPM creation and copr? It'd be great if could collaborate!

**Comment by rashidi77** [comment]
Thank you. I'll take a look at it. I would like to maintain a copr, but 
I don't have  enough knowledge to package rpm files. I tried to package 
one for testing but failed.
